### DISCUSSION TITLE
Lazy refresh [Jun 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6001] [Discord Threads]
> 🙋‍♂️ **trnio**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@959665274024493106> What behaviour are you pointing out at with the term "lazy refresh"?  
#  
> 🙋‍♂️ **trnio**  
#### The table will auto refresh when row data is updated  
#  
> 🙋‍♂️ **trnio**  
#### That is the point I want to say  
#  
> 👨‍🦲 **dstala**  
#### Not supported yet. Page has to be explicitly reloaded to get updated row contents.  
#  
> 🙋‍♂️ **trnio**  
#### So is there any plan to support this feature?  
#  
> 👨‍🦲 **dstala**  
#### Yes. But not in immediate roadmap  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1110764665589485599)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
